app/controllers/concerns/response.rb

```diff
@@ -0,0 +1,5 @@
+module Response
```

Comment:
This was a good approach to keep your controller dry 👍

Reply:
Thanks, @eapenzacharias 😊✌